### PR TITLE
Locale screen

### PR DESCRIPTION
**Pull request by tombettany**
Adds the Locale screen (not visible from the home screen but available with `kano-settings locale`.

Also adds a few useful locale functions

cc @convolu @pazdera @carolineclark

### CODE REVIEW

File: tests/i18n/test_locale.py [hunk 1]
```diff
@@ -221,3 +237,18 @@ def test_set_locale(self):
                     self.assertFalse(True)
```
Comment (convolu) [inline tests/i18n/test_locale.py]: Maybe this would be more appropriate here https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertNotIn

**Comment by convolu** [comment]
Looks good to me

 :+1: :+1: for unit tests. Even though not touched by this commit, there are a few places in the test where a better assert is available. Using it is useful because it allows for better feedback when the test fails.

See https://docs.python.org/2/library/unittest.html#assert-methods for reference

**Comment by tombettany** [comment]
@convolu Although I agree that the asserts could be improved, I'll handle them later so I'll merge this now.

**Comment by convolu** [comment]
@tombettany :+1: :shipit:

**Pull request merged**